graph/path/bench_test.go

```diff
@@ -93,51 +68,40 @@ func abs(a int) int {
 	return a
 }
 
-func benchmarkAStarHeuristic(b *testing.B, g graph.Undirected, h Heuristic) {
-	var expanded int
-	for i := 0; i < b.N; i++ {
-		_, expanded = AStar(simple.Node(0), simple.Node(1), g, h)
-	}
-	if expanded == 0 {
-		b.Fatal("unexpected number of expanded nodes")
+func BenchmarkAStarUndirected(b *testing.B) {
+	benchmarks := []struct {
+		name  string
+		graph graph.Undirected
+		h     Heuristic
+	}{
+		{"GNP Undirected 10 tenth", gnpUndirected_10_tenth, nil},
+		{"GNP Undirected 100 tenth", gnpUndirected_100_tenth, nil},
+		{"GNP Undirected 1000 tenth", gnpUndirected_1000_tenth, nil},
+		{"GNP Undirected 10 half", gnpUndirected_10_half, nil},
+		{"GNP Undirected 100 half", gnpUndirected_100_half, nil},
+		{"GNP Undirected 1000 half", gnpUndirected_1000_half, nil},
+
+		{"NSW Undirected 10 2_2 2", nswUndirected_10_2_2_2, nil},
```

Comment:
Just a nit: the extra underscore in `name`.

Reply:
Fixed